unicodetools/org/unicode/draft/GeneratePickerData2.java

```diff
@@ -162,7 +161,7 @@ public int compare(String o1, String o2) {
     enum MyOptions {
         output(".*", CLDRPaths.BASE_DIRECTORY + "tools/java/org/unicode/cldr/draft/picker/",
             "output data directory"),
-        unicodedata(".*", DraftUtils.UCD_DIRECTORY, "Unicode Data directory"),
+        unicodedata(".*", CLDRPaths.UCD_DATA_DIRECTORY, "Unicode Data directory"),
```

Comment:
This and a few other places was just to update, since DraftUtils was removed from CLDR.